src/RepairManagerAgent/main.py

```diff
@@ -0,0 +1,47 @@
+import requests
+import time
+import yaml
+import logging
+import logging.config
+import os
+
+with open('./logging.yaml', 'r') as log_file:
+    log_config = yaml.safe_load(log_file)
+
+logging.config.dictConfig(log_config)
+logger = logging.getLogger(__name__)
+
+def reboot_node():
+    os.system('sync')
+    os.system('reboot -f')
+
+def Run():
+    while True:
+        etcd_server = os.environ["ETCD_SERVER"]
+        etcd_port = os.environ["ETCD_PORT"]
+        node_name = os.environ["NODE_NAME"]
+        etcd_url = f'http://{etcd_server}:{etcd_port}/v2/keys/{node_name}/reboot'
+        logger.debug(etcd_url)
```

Comment:
You may want to move environment get and this logging out from while. It will not change anyway and will output large chunk of logging.